.travis.yml

```diff
@@ -20,3 +20,15 @@ script:
     - inspekt lint
     - inspekt indent
     - inspekt style
+    - |
+        err=false
+        for file in $(git diff --name-only $TRAVIS_COMMIT_RANGE -- *.py)
+        do
+            echo -n Checking license in file $file...
+            inspekt license --copyright "Copyright" --author "Author" $file || err=true
```

Comment:
Yes, just noticed that as well. I will send a PR for inspekt, adding more license types.